common/app/views/fragments/mostPopularPlaceholder.scala.html

```diff
@@ -11,23 +14,25 @@
         <div class="fc-container__inner">
             <div class="fc-container__header js-container__header">
                 <h2 class="fc-container__header__title">
-                    <a href="@LinkTo(url)" data-link-name="Most viewed @section">Most viewed</a>
+                    <a href="@LinkTo(url)" data-link-name="Most viewed @section">Most popular</a>
                 </h2>
             </div>
-            <div class="fc-container__body fc-container--rolled-up-hide popular-trails">
-                <div class="popular-trails__content js-popular-trails">
+            @defining(Switches.ExtendedMostPopular.isSwitchedOn){ isExtended =>
```

Comment:
Perhaps pass this switch state into the template as a parameter?

Reply:
I've added that to the follow up card too: https://trello.com/c/qETL0JDD/225-get-rid-of-the-old-most-pop-version